docs/sources/userguide/usingdocker.md

```diff
@@ -181,8 +181,8 @@ Our Python application is live!
 > **Note:**
 > If you have used the boot2docker virtual machine on OS X, Windows or Linux,
 > you'll need to get the IP of the virtual host instead of using localhost.
-> You can do this by running the following in
-> the boot2docker shell.
+> You can do this by running the following outside of
+> the boot2docker shell (i.e. your command line or terminal application).
```

Comment:
Nice catch. Thanks. 
Sorry to nit pick, but you need a comma after "i.e.". We might also make things a little clearer by rearranging the noun strings: (i.e., from your terminal or command-line application). Just a suggestion... thanks for the contribution.